cmd/mattermost/commands/server.go

```diff
@@ -19,7 +14,14 @@ import (
 	"github.com/mattermost/mattermost-server/wsapi"
 	"github.com/mattermost/viper"
 	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/spf13/cobra"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
```

Comment:
This rearrangement of imports might cause merge conflicts.

Reply:
removed the rearrange.